administrator/components/com_joomlaupdate/views/update/tmpl/finaliseconfirm.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_joomlaupdate
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+JHtml::_('behavior.keepalive');
+JHtml::_('bootstrap.tooltip');
+JHtml::_('jquery.framework');
+
+JLoader::register('ModLoginHelper', JPATH_ADMINISTRATOR . '/modules/mod_login/helper.php');
+
+$twofactormethods = ModLoginHelper::getTwoFactorMethods();
```

Comment:
Let it be this way if you create a joomlaupdate package it works in ollder version too